src/btree/bt_sync_obsolete.c

```diff
@@ -94,11 +105,41 @@ __sync_obsolete_inmem_evict(WT_SESSION_IMPL *session, WT_REF *ref)
         /* Mark the obsolete page to evict soon. */
         __wt_page_evict_soon(session, ref);
         WT_STAT_CONN_DSRC_INCR(session, checkpoint_cleanup_pages_evict);
```

Comment:
Sometimes we use the `cc` prefix and sometimes `checkpoint_cleanup` for stats, we should be consistent.

Reply:
For all stats, we use the `checkpoint_cleanup` and not `cc`.